tools/verify_course_repo.js

```diff
@@ -0,0 +1,10 @@
+const courseDB = require('../sync/course-db');
+(async () => {
+    const courseData = await courseDB.loadFullCourseNew('/course');
+    const errors = [];
+    courseDB.writeErrorsAndWarningsForCourseData('fakeCourseId', courseData, line => line ? errors.push(line) : null);
```

Comment:
Nit: the course directory argument is currently unused and should probably be removed from the function. In the meantime, let's just pass null.

```suggestion
    courseDB.writeErrorsAndWarningsForCourseData(null, courseData, line => line ? errors.push(line) : null);
```

Aside: is there a case when `line` is null? I looked at the code again and couldn't see why this check would be necessary.